tasks/config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
     components: 'components/**/*.*',
     templates: [
       '{templates, dummy}/**/*.html',
-      '!templates/e-commerce/partials/hero/*.html'
+      '!templates/e-commerce/partials/hero/*.html',
+      '!templates/lune/partials/**/*.html'
```

Comment:
lets make this a general pattern , anything in partials should not be processed as templates or validated.